SwiftCharts/Layers/ChartPointsTouchHighlightLayer.swift

```diff
@@ -18,13 +18,16 @@ open class ChartPointsTouchHighlightLayer<T: ChartPoint, U: UIView>: ChartPoints
     fileprivate let chartPointLayerModelForScreenLocFilter: ChartPointLayerModelForScreenLocFilter
 
     open let longPressGestureRecognizer: UILongPressGestureRecognizer
+    
+    open var fadeAnimationCompletion: (()->())?
```

Comment:
The naming should be consistent, call the parameter as well as the variable `highlightCompletion`. To avoid compiler error use `self`: `self.highlightCompletion = highlightCompletion`. Lastly I have been using in this library "on<X>" naming convention, e.g. `onComplete`, `onTap`, etc. So to keep it consistent we should call this `onCompleteHighlight`.